src/router.php

```diff
@@ -2,25 +2,51 @@
 
 namespace Bugcache;
 
-use Aerys as aerys;
+use Aerys;
```

Comment:
I'd rather stick with all namespaces uppercase. I don't want to alias every imported namespace. And it should be consistent.